### DISCUSSION TITLE
podman-compose

### DISCUSSION THREAD

**Discussion by bskou57** [ansibleguy76/ansibleforms#203] [Q&A] [answered]
Hi

Can we use podman-compose instead of docker-compose ?

Thanks for your support

**Comment by ansibleguy76** [comment]
i know people have succeeded with it.  i will investigate and extend the manual to podman.

**Comment by birb57** [comment]
test it and seems working fine

thanks for your support

**Comment by ansibleguy76** [comment]
with podman-compose?  were changes needed?

**Comment by ansibleguy76** [comment, marked as answer]
I tested podman this week.  Ubuntu was hell.  Version with 3.4 and had bugs.  I had to move to debian.  There it works like a charm (podman and podman-compose).

**Comment by bskou57** [comment]
Hi

working fine with podman compose on rhel8


Thanks
